### ISSUE TITLE
Canonical URL for TAG findings should not default to /TR/ space

### ISSUE THREAD

**Issue by ylafon** [w3c/respec#3786]
## Important info

- URL to affected spec: https://w3ctag.github.io/ethical-web-principles/ 
- ReSpec version: ReSpec 26.13.4
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

Doing an export as HTML, it produces 
`<link rel="canonical" href="https://www.w3.org/TR/ethical-web-principles/">`
instead of 
`<link rel="canonical" href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">`
Basically, the base URL should be `https://www.w3.org/2001/tag/doc/` instead of `https://www.w3.org/TR/` for TAG findings.
Cheers,

**Comment by marcoscaceres** [comment]
Bug indeed for TAG things... 

You can override it:
https://respec.org/docs/#canonicalURI

Will send you a PR and update the code that generates the URL for findings.
